utilities/lua/rocks_lua_compaction_filter.cc

```diff
@@ -0,0 +1,240 @@
+//  Copyright (c) 2016, Facebook, Inc.  All rights reserved.
+//  This source code is licensed under the BSD-style license found in the
+//  LICENSE file in the root directory of this source tree. An additional grant
+//  of patent rights can be found in the PATENTS file in the same directory.
+
+#if defined(LUA) && !defined(ROCKSDB_LITE)
```

Comment:
Why do we need to do that ? should not we disable LUA if ROCKSDB_LITE is enabled ?